src/vs/platform/environment/node/environmentService.ts

```diff
@@ -114,6 +114,9 @@ export class EnvironmentService implements IEnvironmentService {
 	@memoize
 	get sharedIPCHandle(): string { return `${getIPCHandlePrefix()}-${pkg.version}-shared${getIPCHandleSuffix()}`; }
 
+	@memoize
+	get nodeCachedDataDir(): string { return path.join(this.userDataPath, 'CachedData', pkg.version); }
```

Comment:
Well, nevermind, since we update insiders nightly, the cache needs to be invalidated anyways.